dist/react-filterbar.js

```diff
@@ -16951,13 +16951,21 @@ var FilterBarActor = /*#__PURE__*/function () {
   }, {
     key: "loadSavedSearch",
     value: function loadSavedSearch(searchId) {
+      var _this = this;
+
       this.disableAllFilters();
       var savedSearch = this.filterBarStore.getSavedSearch(searchId);
       var filters = JSON.parse(savedSearch.configuration);
 
       if (this.verifySavedFilters(filters)) {
-        for (var filter in filters) {
-          this.enableFilter(filter, filters[filter]);
+        if (filters instanceof Array) {
+          filters.forEach(function (filter) {
+            return _this.enableFilter(filter.uid, filter.value);
```

Comment:
for example here if we use
```suggestion
            return component.enableFilter(filter.uid, filter.value);
```
that's more readable IMHO

Reply:
Oh ... if it's just auto-generated dist file, please ignore it 🙈